data/speedtest

```diff
@@ -1,3 +1,4 @@
 # This tag 'speedtest' is for backward compatibility of 'geosite:speedtest' in V2Ray.
 
+include:dslreports
```

Comment:
# This tag 'speedtest' is for backward compatibility of 'geosite:speedtest' in V2Ray.